src/ARCtrl/ARCtrl.fs

```diff
@@ -355,6 +356,9 @@ type ARC(?isa : ISA.ArcInvestigation, ?cwl : CWL.CWL, ?fs : FileSystem.FileSyste
         |> Option.bind (fun tree -> if ignoreHidden then tree |> FileSystemTree.filterFolders (fun n -> not (n.StartsWith("."))) else Some tree)
         |> Option.defaultValue (FileSystemTree.fromFilePaths [||])
         
+    static member DefaultContracts = Map<string,Contract> [|
```

Comment:
Needed in some function or just for user convenience? And if so, why map when the Contract does contain the path itself?

Reply:
This is api user convinience and a map so the contract is easier to find? That part is up for discussion